mobile/src/hooks/use-search.ts

```diff
@@ -0,0 +1,28 @@
+import { useGno } from "./use-gno";
+
+const MAX_RESULT = 100;
```

Comment:
I most other search tools I've seen, the number of results is smaller and fits on one screen, like 10.